exchanges/kraken/kraken_websocket.go

```diff
@@ -974,65 +990,74 @@ func (k *Kraken) GenerateAuthenticatedSubscriptions() ([]stream.ChannelSubscript
 
 // Subscribe sends a websocket message to receive data from the channel
 func (k *Kraken) Subscribe(channelsToSubscribe []stream.ChannelSubscription) error {
-	var subs []WebsocketSubscriptionEventRequest
+	var subscriptions = make(map[string]*[]WebsocketSubscriptionEventRequest)
```

Comment:
I'm not sure why this is a pointer slice given that it's not being passed around to other functions and expected to have sizing changes

Reply:
I just wanted to make sure that if we exceed the capacity with append and a new array is created it's still referenced in the map itself. Before it was just your standard scoped slice.